src/containers/app/index.js

```diff
@@ -129,22 +127,23 @@ class App extends React.Component {
                             {showDash ? (
                                 <TopNav web3Connect={this.props.web3Connect} />
                             ) : (
-                                    <AppMenu
-                                        web3Connect={this.props.web3Connect}
-                                        location={this.props.location}
-                                        showMenu={this.state.showMobileMenu}
-                                        toggleMenu={this.toggleMenu}
-                                    />
-                                )}
+                                <AppMenu
+                                    web3Connect={this.props.web3Connect}
+                                    location={this.props.location}
+                                    showMenu={this.state.showMobileMenu}
+                                    toggleMenu={this.toggleMenu}
+                                />
+                            )}
                             {showDash ? <SideNav /> : null}
 
-
                             <div className={showDash ? "Site-content Site-content__dash" : "Site-content"}>
-                                <EthereumTxStatus />
-                                <LegacyLoanManagers />
-                                <LegacyLockers />
-                                <LegacyExchanges />
-                                <LegacyTokens />
+                                <div className={hideMsgPanel && " hideMsgPanel"}>
```

Comment:
let's not render hidden elements in the DOM! You can conditionally render these elements depending on `hideMsgPanel` or as suggested above using `showDash`.